pkgs/applications/graphics/gltfpack/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, nixos
+, cmake
+, meshoptimizer
+, gltfpack
+}:
+let basis_universal = fetchFromGitHub {
+      owner = "zeux";
+      repo = "basis_universal";
+      rev = "8903f6d";
+      hash = "sha256-o3dCxAAkpMoNkvkM7qD75cPn/obDc/fJ8u7KLPm1G6g=";
+    };
+in
+stdenv.mkDerivation rec {
+  name = "gltfpack";
+  version = "0.18";
+  src = fetchFromGitHub {
+    owner = "zeux";
+    repo  = "meshoptimizer";
+    hash  = "sha256-OWeptdnKFvTyfkz0sFCpiTI7323GfVE8vb8bNUBnslA=";
+    rev   = "49d9222";
```

Comment:
Please use `rev = "refs/tags/v${version}"` instead. This makes updating easier.  Same for the other package.

Reply:
I did, but it wouldn't compile w/the binary output enabled when i used a specific tag.

Reply:
Is it acceptable to downgrade the version to 0.17? I can't actually get the 0.18 to build reproducibly w/a binary output on my machine.

Reply:
I can confirm, that the v0.18 version does not compile. Unfortunately, the v0.18 version is very old and no new release is available. https://github.com/zeux/meshoptimizer/compare/v0.18...49d9222 . Maybe we should set the `version` attribute to `"unstable-2023-03-22"` then, because that is the day of the 49d9222 commit.

Reply:
> Is it acceptable to downgrade the version to 0.17? I can't actually get the 0.18 to build reproducibly w/a binary output on my machine.

We don't have to downgrade. We can use this commit with the unstable version attribute.